templates/ingress/ingress.yaml

```diff
@@ -89,6 +91,12 @@ spec:
         backend:
           serviceName: {{ template "harbor.core" . }}
           servicePort: 80
+      {{- if eq .Values.expose.ingress.controller "alb" }}
```

Comment:
What is it used for?

Reply:
In ALB ingress controller, path matching rules are applied in order from the top.  "/" seems to be recognized as "/ *", so rules after "/" will not work.

Reply:
How about put the portal section as the last one? this should work for other ingress controllers